test-data/unit/check-classes.test

```diff
@@ -1088,7 +1088,7 @@ reveal_type(Foo().Meta.name)  # N: Revealed type is "builtins.str"
 
 class A:
     def __init__(self):
-        self.x = None # type: int
+        self.x = None # type: int  # N: By default the bodies of functions without signature annotations are not type checked, consider using "--check-untyped-defs"
```

Comment:
This is a special case, mypy does use type annotations in unannotated `__init__()`. So I would propose to not show this note in `__init__()`.

Reply:
Actually there still may be type errors like `self.x: int = "no"`, so withdraw this comment.